themes/gauge_theme/_landing.html

```diff
@@ -132,7 +132,7 @@ <h3>Troubleshooting Guide</h3>
     <li class="gauge-features__feature">
       <div class="feature-wrapper">
         <a href="//taiko.gauge.org/#integrating-gauge">
-          <h3>Gauge with Taiko</h3>
+          <div class="feature-header">Gauge with Taiko</div>
           <p class="feature-desc">
             Use Taiko to create test scripts and integrate with Gauge specs.
           </p>
```

Comment:
I think we should change the class name to `feature-heading` as the elements for which we have used this class represents a heading element not [`header`](https://www.w3schools.com/tags/tag_header.asp) element.